src/09-coveio_abi.adoc

```diff
@@ -686,3 +686,144 @@ The possible error codes returned in `sbiret.error` are shown below.
 | SBI_ERR_ALREADY_STOPPED | The device interface is already stopped.
 | SBI_ERR_FAILED          | The operation failed for unknown reasons.
 |===
+
+=== CoVE IO TSM Extension (EID #0x434F5654 "COVT")
+
+==== PCIe Root Port IDE Management
+
```

Comment:
Add a `sbi_covt_rp_ide_key_status()` call to get a key status: unknown, programmed or enabled (programmed implied not enabled).